core/mixins/_generate-percentage-classes-at-breakpoints.scss

```diff
@@ -34,6 +34,7 @@ $mixin-numerators:            (one two three four five six seven eight nine
 $mixin-denominator-words:     (whole half third quarter fifth sixth seventh
                               eighth ninth tenth eleventh twelfth) !default;
 
+$apply-percentage-classnames: false !default;
```

Comment:
Can you follow the standard of prefixing mixin settings with `$mixin-`? So: `$mixin-apply-percentage-classnames`.

Ta